besom-cfg/k8s/src/main/scala/ConfiguredContainerArgs.scala

```diff
@@ -0,0 +1,172 @@
+package besom.cfg.k8s
+
+import besom.cfg.internal.*
+import besom.types.{Input, Context, Output}
+import besom.cfg.*
+import besom.json.*
+import besom.cfg.containers.*
+import besom.api.kubernetes.core.v1.inputs.*
+
+import scala.util.*
+import scala.quoted.*
+import besom.cfg.k8s.syntax.*
+
+// this is besom-cfg-kubernetes entrypoint
```

Comment:
since this is the entrypoint, I'd strongly advice adding even the most barebones scaladoc :)

Reply:
I'll strongly consider that once besom-cfg is beyond a prototype